### DISCUSSION TITLE
UseLowerCaseLabels not working

### DISCUSSION THREAD

**Discussion by kasparas92** [Gremlinq/ExRam.Gremlinq#1473] [Q&A] [answered]
Hello,

I'm trying to refactor code with ExRam.Gremlinq package instead of Gremlin.Net. We have existing GraphDb, which we want to reuse. All our vertices has Labels in lowercases. I did found extension method UseLowerCaseLabels, which can be used to change Labels implementation. This configuration is not working. When I add vertex in GraphDB, Label value is still in PascalCase. Code snippet for configuration attached. Did you do something wrong?

`_g = g
.ConfigureEnvironment(env => env.UseModel(GraphModel.FromBaseTypes<Vertex, Edge>()
    .ConfigureElements(x => x.UseLowerCaseLabels())))
.UseCosmosDb<Vertex, Edge>(configurator => configurator
     .At(new Uri("uri"))
     .OnDatabase("dev")
     .OnGraph("Graph1")
     .WithPartitionKey(x => x.Affinity!)
     .AuthenticateBy("auth_key")
     .UseNewtonsoftJson());`

**Comment by danielcweber** [comment, marked as answer]
Don't call `UseModel` before `UseCosmosDb`. Instead, use `ConfigureModel` _after_ `UseCosmosDb`.

**Comment by kasparas92** [comment]
Thanks @danielcweber it worked.
